IDEAS/Buildings/Components/Shading/OverhangAndScreen.mo

```diff
@@ -44,6 +44,10 @@ initial equation
 </html>", revisions="<html>
 <ul>
 <li>
+July 18, 2022 by Filip Jorissen:<br/>
+Refactored for #1270 for including thermal effect of screens.
+</li>
+<li>
```

Comment:
Only change in this file is the revision history. Did I miss a change, or should this revision be added to another model?